eos-live-boot-generator

```diff
@@ -58,3 +58,20 @@ Type=oneshot
 ExecStart=/bin/bash -c 'echo -e "[Context]\nfilesystems=/run/media/eoslive/.kolibri;\n\n[Environment]\nKOLIBRI_HOME=/run/media/eoslive/.kolibri" > /run/flatpak/overrides/org.learningequality.Kolibri'
 SETUP_KOLIBRIHOME
 ln -sf "$unit_path" "$multi_user_target_wants/"
+
+unit_path="$dest_dir/eoslive-encyclopedia-setup.service"
```

Comment:
Maybe to be consistent you should call this `flatpak-extension-kiwix-usb-content-setup.service`

Reply:
Perhaps – I picked it to be consistent with the unit above, and to emphasise the fact that it's specific to live systems.